_field-types/mapping-parameters/store.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: default
+title: Store
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 180
+has_children: false
+has_toc: false
+---
+
+# Store
+
+The `store` mapping parameter determines whether the value of a field should be stored separately from the `_source` and made directly retrievable using the `stored_fields` option in a search request.
+
+By default, `store` is set to `false`, meaning field values are not stored individually and are only available as part of the `_source` document. If `store` is set to `true`, you can disable the `_source` to save disk space and still retrieve specific fields.
+
```

Comment:
In line 15, could you link to retrieve specific fields? https://opensearch.org/docs/latest/search-plugins/searching-data/retrieve-specific-fields/

Reply:
Thats done now